tests/test_bolt2-30-channel_type-open-accept-tlvs.py

```diff
@@ -44,13 +44,10 @@ def test_open_channel(runner: Runner, with_proposal: Any) -> None:
         Block(blockheight=102, txs=[tx_spendable]),
         Connect(connprivkey="02"),
         ExpectMsg("init"),
-        TryAll(
-            # BOLT-a12da24dd0102c170365124782b46d9710950ac1 #9:
-            # | 20/21 | `option_anchor_outputs`          | Anchor outputs
-            Msg("init", globalfeatures="", features=bitfield(13, 21)),
-            # BOLT #9:
-            # | 12/13 | `option_static_remotekey`        | Static key for remote output
-            Msg("init", globalfeatures="", features=bitfield(13)),
+        Msg(
+            "init",
+            globalfeatures=runner.runner_features(globals=True),
+            features=runner.runner_features(additional_features=[12]),
```

Comment:
This doesn't seem right, and you are changing the meaning of the test here. The previous value was 13, so you should keep 13 because it will tell the node that the feature is optional.

Question for you here: What happens if the `runner_features` adds feature 12? and the test defined the feature 13?

You now have the feature required and optional, and it is a mess.

There is no definition in the BOLT 9 of this needs to happens here

Reply:
I must have mis typed it here. I will fix this.